frontend/src/components/viewers/Tensorboard.tsx

```diff
@@ -56,6 +64,10 @@ class TensorboardViewer extends Viewer<TensorboardViewerProps, TensorboardViewer
     this._checkTensorboardApp();
   }
 
+  public onChangeFunc(e: React.ChangeEvent<{ name?: string; value: unknown }>): void {
+    this.setState({ tensorflowVersion: e.target.value as string });
```

Comment:
typecasting hides problems later.
You can do the following in type safe way.
```
if (typeof e.target.value !== 'string') {
  return; // or report a warning, it's fine
}
this.setState({ tensorflowVersion: e.target.value });
```

Reply:
Thanks! Will modify it.

Reply:
@dldaisy Reminder of this